rome-modules/src/main/java/com/rometools/modules/atom/modules/AtomLinkModule.java

```diff
@@ -13,16 +13,19 @@
  * limitations under the License.
  *
  */
+
 package com.rometools.modules.atom.modules;
 
 import com.rometools.rome.feed.atom.Link;
 import com.rometools.rome.feed.module.Module;
 
+import java.util.List;
+
 public interface AtomLinkModule extends Module {
 
-    public static final String URI = "http://www.w3.org/2005/Atom";
+    String URI = "http://www.w3.org/2005/Atom";
 
-    public Link getLink();
+    List<Link> getLinks();
```

Comment:
This is not backward compatible, maybe keep both `get/setLinks()` and deprecated `get/setLink()`?